scripts/release/add_notes_changelog.sh

```diff
@@ -0,0 +1,95 @@
+#!/bin/bash
```

Comment:
right now I expect the ones creating release branches to execute this script @Eugene-hu 

I've added this to the [RELEASE_GUIDELINES.md]() document